Compiler warning on NixOS

Every time I compile rust-simplicity I get this long warning.

```
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/bitstream.h:6,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/bitstream.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/dag.h:8,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/dag.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/bitstream.h:6,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/deserialize.h:6,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/deserialize.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/bounded.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/eval.h:6,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/eval.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/sha256.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/frame.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/frame.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/sha256.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/frame.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/jets.h:7,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/jets.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/sha256.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/frame.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/jets.h:7,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/jets-secp256k1.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/limits.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include-fixed/limits.h:203,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include-fixed/syslimits.h:7,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include-fixed/limits.h:34,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/rsort.h:4,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/rsort.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/sha256.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/sha256.c:8:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/bounded.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/type.h:6,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/type.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/dag.h:8,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/typeInference.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/typeInference.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/include/simplicity/elements/env.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/env.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/../../sha256.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/ops.h:6,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/ops.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/include/simplicity/elements/exec.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/exec.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/../../sha256.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/../../frame.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/../../jets.h:7,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/jets.h:6,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/jets.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/../../sha256.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/primitive.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/primitive.c:3:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/ctx8Pruned.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/ctx8Pruned.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/ctx8Unpruned.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/ctx8Unpruned.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/hashBlock.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/hashBlock.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/schnorr0.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/schnorr0.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/schnorr6.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/schnorr6.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/checkSigHashAllTx1.h:5,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/primitive/elements/checkSigHashAllTx1.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdint.h:26,
warning: simplicity-sys@0.1.0:                  from /nix/store/hf2gy3km07d5m0p1lwmja0rg9wlnmyr7-gcc-12.3.0/lib/gcc/x86_64-unknown-linux-gnu/12.3.0/include/stdint.h:9,
warning: simplicity-sys@0.1.0:                  from depend/simplicity/bitstream.h:6,
warning: simplicity-sys@0.1.0:                  from depend/wrapper.c:3:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
warning: simplicity-sys@0.1.0: In file included from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/bits/libc-header-start.h:33,
warning: simplicity-sys@0.1.0:                  from /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/stdlib.h:26,
warning: simplicity-sys@0.1.0:                  from depend/env.c:1:
warning: simplicity-sys@0.1.0: /nix/store/mrgib0s2ayr81xv1q84xsjg8ijybalq3-glibc-2.38-27-dev/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
warning: simplicity-sys@0.1.0:   414 | #  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
warning: simplicity-sys@0.1.0:       |    ^~~~~~~
```

--- a/simplicity-sys/build.rs
+++ b/simplicity-sys/build.rs
@@ -44,6 +44,7 @@ fn main() {
     cc::Build::new()
         .std("c11")
         .flag_if_supported("-fno-inline-functions")
+        .opt_level(2)
         .files(files)
         .files(test_files)
         .file(Path::new("depend/wrapper.c"))

